internal/core/services/worker/manager/workers_manager.go

```diff
@@ -20,7 +20,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-package workers_manager
+package manager
```

Comment:
Hey @guilhermocc how you name the package as `worker`, what do you think to name the struct as `Manager`, so when the client uses this package it will call `workers.Manager`?